scripts/image_maniplation/size insitutions_images_for_circle.py

```diff
@@ -0,0 +1,50 @@
+from PIL import Image
+import math
+import os
+
+#This script enlarges institions shield imamges so the corners show when they are displayed as circles.
+
+INSTITUTIONS_IMG_PATH =  '../website/static/img/institutions/'
+
+def pythagonreanTheorem(x,y):
+    return math.ceil((x**2 + y**2)**.5)
+
+def resize(img):
+    newSize = int(pythagonreanTheorem(img.size[0],img.size[1])) # new corners within circle
+    newImg = Image.new('RGBA', (newSize,newSize), 'white')
+    newImg.paste(img, (newSize/2-img.size[0]/2,newSize/2-img.size[1]/2)) #center in larger img
+    return newImg
+
+def alphaToWhite(img):
+    pixeldata = list(img.getdata())
+    for i,pixel in enumerate(pixeldata):
+        if len(pixel) == 4:
+            if pixel[3] == 0:
```

Comment:
How does this work with images or logos that may have non-zero opacity? i.e. if it's semi-transparent?